src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -169,6 +175,31 @@ export const View = (props: Props) => {
         </div>
       </section>
 
+      <section className={styles.socialProofWrapper}>
+        <h2>
+          {props.l10n.getString("landing-all-social-proof-title", {
+            num_users: 10,
+          })}
+        </h2>
+        <p className={styles.socialProofDescription}>
+          {props.l10n.getString("landing-all-social-proof-description", {
+            num_countries: 237,
+          })}
+        </p>
+        <div className={styles.pressLogos}>
+          <p className={styles.label}>
+            {props.l10n.getString("landing-all-social-proof-press")}
+          </p>
+          <Image src={ForbesLogo} alt="" />
+          <Image src={TechCruchLogo} alt="" />
+          <Image src={PCMagLogo} alt="" />
+          <Image src={CNETLogo} alt="" />
+          <Image src={GoogleLogo} alt="" />
```

Comment:
Since these images _do_ convey information, I think we should use the company names in the alt texts?

(@flodolo - since these are names, do they need to be localised, or can we just hardcode them?)